nextcord/abc.py

```diff
@@ -523,6 +531,14 @@ def permissions_synced(self) -> bool:
         category = self.guild.get_channel(self.category_id)
         return bool(category and category.overwrites == self.overwrites)
 
+    @property
+    def jump_url(self) -> str:
+        """:class:`str`: Returns a URL that allows the client to jump to the referenced messageable.
```

Comment:
```suggestion
        """:class:`str`: Returns a URL that allows the client to jump to this channel.
```
gonna just commit myself to not bother you